securedrop/tests/functional/source_navigation_steps.py

```diff
@@ -123,6 +123,22 @@ def submit_page_loaded():
 
         self.wait_for(submit_page_loaded)
 
+    def _source_chooses_to_submit_documents_with_colliding_journalist_designation(self):
```

Comment:
If you use the existing convenience methods like `self._source_chooses_to_submit_documents()` in `TestSourceInterfaceDesignationCollision`, instead of adding this, we won't have another place that needs updating when page content changes.